packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCheck.tsx

```diff
@@ -38,6 +38,7 @@ interface LivenessCheckProps {
   streamDisplayText: Required<StreamDisplayText>;
   errorDisplayText: Required<ErrorDisplayText>;
   components?: FaceLivenessDetectorComponents;
+  beginLivenessCheck: () => void;
```

Comment:
Does this new prop affect customers, or is this only used internally?

Reply:
only used internally